src/main/scala/karazin/scala/users/group/week2/homework/Homework.scala

```diff
@@ -55,7 +73,15 @@ object Homework:
 
     private lazy val g = gcd(abs(x), y)
 
-    override def equals(other: Any): Boolean = ???
+    override def equals(other: Any): Boolean =  {
```

Comment:
It's very important to follow the contract on `equals` and `hashCode`. Every time `equals` is changed, `hashCode` should be adjusted as well.

Please review this article: https://www.baeldung.com/java-equals-hashcode-contracts